5.0/debian-10/rootfs/opt/bitnami/scripts/libredis.sh

```diff
@@ -240,8 +240,12 @@ redis_validate() {
 #########################
 redis_configure_replication() {
     info "Configuring replication mode"
-
-    redis_conf_set replica-announce-ip "$(get_machine_ip)"
+    
+    if [[ -n "$REDIS_REPLICA_IP" ]]; then
+      redis_conf_set replica-announce-ip "${REDIS_REPLICA_IP}"
+    else
+      redis_conf_set replica-announce-ip "$(get_machine_ip)"
+    fi
```

Comment:
This can be simplified by setting a default value for the variable. What do you think?

```suggestion

    redis_conf_set replica-announce-ip "${REDIS_REPLICA_IP:-$(get_machine_ip)}"
```